docs/packages/connect/methods/authorizeCoinJoin.md

```diff
@@ -42,7 +42,7 @@ const result = await TrezorConnect.authorizeCoinJoin(params);
 TrezorConnect.authorizeCoinJoin({
     path: "m/10086'/0'/0'",
     maxRounds: 3,
-    maxCoordinatorFeeRate: 5000000, // 5% => 0.05 * 10**8;
+    maxCoordinatorFeeRate: 500000, // 5% => 0.005 * 10**8;
```

Comment:
The comment is incorrect. For 500000 it should say `0.5% => 0.005 * 10**8;`. (The old comment was also incorrect.)

Reply:
Right, I fixed that.